src/rdkafka.h

```diff
@@ -4827,8 +4861,24 @@ struct rd_kafka_group_member_info {
         void *member_assignment;    /**< Member assignment (binary),
                                      *    format depends on \p protocol_type. */
         int member_assignment_size; /**< Member assignment size in bytes */
+        rd_kafka_topic_partition_list_t
```

Comment:
Public types may not be altered since it would break the ABI compatibility guarantees, so you will probably need to add a new Admin type here that is private.

Reply:
reverted rd_kafka_group_member_info to previous state